src/updateAttribute.js

```diff
@@ -49,6 +49,29 @@ function setAttribute (node, attrName, attrValue, oldAttrValue) {
     } else {
       // if attribute is value property
       node[attrName] = attrValue;
+
+      //Remove property from new Attribute
+      if ( typeof oldAttrValue != "undefined" ) {
+           var old_keys = Object.keys( oldAttrValue );
+           let old_iterval = old_keys.length;
```

Comment:
The counter name seems incorrect. It's not an interval. It's incrementer/counter.